modules/porous_flow/doc/content/modules/porous_flow/boundaries.md

```diff
@@ -38,6 +38,7 @@ This basic sink boundary condition is implemented in [`PorousFlowSink`](PorousFl
 The basic sink may be multiplied by a MOOSE Function of the pressure
 of a fluid phase *or* the temperature:
 \begin{equation}
+  \label{eq:s_f_g}
```

Comment:
PLEASE CHANGE the definition of fluid mobility!!  This is nothing to do with your additions - i just noticed it is wrong.  Ooops.  It should NOT contain the relative permeability.  So please remove k_r from the definition of mobility in this PR.

Reply:
Done

Reply:
I think that the relative permeability should be retained in Eq 3 and in the paragraph beginning with "The numerical value of the conductance..." Is that correct?

Reply:
Yes, k_r should appear in those two places, you are correct.   It should just be removed from the definition of "Fluid mobility" in the second bullet point nearer the top of the page.  Thanks :-)